### DISCUSSION TITLE
How many 🙂🙃 do you see?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1214] [Galaxy Trivia] [answered]
<img width="351" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/277b57e6-3f63-4a35-96dd-b33c2bb8ab73">

**Comment by peterbaccaro** [comment]
I can see **50** happy and sad faces.

**Comment by CedericRomeyns** [comment, marked as answer]
50

**Reply by ParanoidUser** [reply]
I hope you really found all of them 😄.
